fearless/Modules/Staking/StakingAmount/ViewModel/RewardDestinationViewModelFactory.swift

```diff
@@ -72,21 +72,24 @@ final class RewardDestinationViewModelFactory: RewardDestinationViewModelFactory
     ) -> LocalizableResource<RewardDestinationViewModelProtocol> {
         LocalizableResource { locale in
             let amountFormatter = amountFormatter.value(for: locale)
-            let percentageFormatter = NumberFormatter.percent.localizableResource().value(for: locale)
+            let percentageAPYFormatter = NumberFormatter.percentAPY.localizableResource().value(for: locale)
```

Comment:
I suggest to move NumberFormatter.percentAPY outside of the closure, so we just call value(for:) inside